sahi/utils/mot.py

```diff
@@ -232,11 +233,57 @@ def _create_info_file(self, seq_length: int, export_dir: str):
             file.write(f"imWidth={self.image_width}\n")
             file.write(f"imHeight={self.image_height}")
 
+    def _create_frame_symlinks(self, coco_images_dir: str, export_dir: str):
+        """
+        Args:
+            coco_images_dir (str): Image directory of coco data to be converted.
+            export_dir (str): Symlink directory that will contain symbolic links
+                              pointing to coco image files.
+        """
+
+        i = 1
+
+        img1 = Path(os.path.abspath(export_dir)) / "img1/"
+        img1.mkdir(parents=True, exist_ok=True)
+
+        for mot_frame in self.frame_list:
+            if Path(mot_frame.file_name).suffix == "":
+                print(f"image file has no suffix, skipping it: '{mot_frame.file_name}'")
+                return
+            elif Path(mot_frame.file_name).suffix not in [".jpg", ".jpeg"]:  # TODO: extend this list
+                print(f"image file has incorrect suffix, skipping it: '{mot_frame.file_name}'")
+                return
+            # set coco and mot image paths
+            coco_img_path_tmp = os.path.join(coco_images_dir, mot_frame.file_name)
+            suffix = Path(mot_frame.file_name).suffix
+
+            if Path(coco_img_path_tmp).is_file():
+                coco_image_path = os.path.abspath(coco_img_path_tmp)
+            else:
+                assert coco_images_dir is not None, (
+                    "You have to specify image_dir " "of Coco object for mot conversion."
```

Comment:
update this assertion as 
"\`file_name\` of mot_frames is not an absolute path. You have to specify \`images_dir\` for mot conversion."